lib/registries/addon/components/registries-navbar/component.ts

```diff
@@ -32,6 +32,7 @@ export default class RegistriesNavbar extends AuthBase {
     @service features!: Features;
 
     provider?: RegistrationProviderModel;
+    defaultProviderId: string = 'osf';
```

Comment:
Should we just go all the way with this and put it in settings? It feels like a thing that would be in settings.